editor/plugins/tiles/tiles_editor_plugin.h

```diff
@@ -97,11 +114,15 @@ class TilesEditorPlugin : public EditorPlugin {
 
 	// To synchronize the atlas sources lists.
 	void set_sources_lists_current(int p_current);
-	void synchronize_sources_list(Object *p_current);
+	void synchronize_sources_list(Object *p_current, Object *p_current2);
```

Comment:
Those arguments should be renamed. Something like `p_current_list` and `p_current_sort_button`